brazil.ion

```diff
@@ -0,0 +1,33 @@
+'brazil_package_spec@1.0'
```

Comment:
Should we be publishing brazil files?

Reply:
Following what's done for RSW. No sensitive information in the file.